added ci/cd example for github actions (#31843)
DataDog/documentation: content/en/llm_observability/experiments/_index.md

   print(f"View experiment: {experiment.url}")
   ```

## Setting up an automated experiment in CI/CD
You can run an `experiment` manually or configure it to run automatically in your CI/CD pipelines. For example, run it against your dataset on every change to compare results with your baseline and catch potential regressions.

### GitHub Actions
Use the following GitHub Actions workflow as a template to run an experiment automatically whenever code is pushed to your repository.

> - Workflow files live in the `.github/workflows` directory at the root of your repository.
> - Workflow files use YAML syntax and use the `.yml` extension.

Reviewer: ```suggestion
**Note**: Workflow files live in the `.github/workflows` directory and must use YAML syntax with the `.yml` extension.”
```
